dashboard/app/models/frequency_analysis.rb

```diff
@@ -0,0 +1,13 @@
+class FrequencyAnalysis < Widget
```

Comment:
If possible, we should deploy this model before the `Frequency Analysis.level` file (it can even be an empty class).  Otherwise we may see errors on frontends when the database contains rows with `type: 'FrequencyAnalysis'` but no corresponding model.